include/deal.II/base/vectorization.h

```diff
@@ -224,6 +224,16 @@ class VectorizedArray
   // POD means that there should be no user-defined constructors, destructors
   // and copy functions (the standard is somewhat relaxed in C++2011, though).
 
+  VectorizedArray() = default;
```

Comment:
I suggest we add a short comment `Default empty constructor, leaving the data in an uninitialized state similar to float/double.` to this new constructor as well.

Reply:
@kronbichler: It is pushed. @masterleinad: Hope I did miss any this time!